models/user.js

```diff
@@ -0,0 +1,15 @@
+'use strict';
+
+const userDefinition = (sequelize, DataTypes) => {
+  const User = sequelize.define('User', {
+    firstName: DataTypes.STRING,
+    lastName: DataTypes.STRING,
+    email: DataTypes.STRING
```

Comment:
Email should be unique too not just string. Read the documentation on sequelize to have a better understanding.